src/windows/process.rs

```diff
@@ -225,17 +225,6 @@ fn windows_8_1_or_newer() -> &'static bool {
     })
 }
 
-#[cfg(feature = "debug")]
-unsafe fn display_ntstatus_error(ntstatus: windows::core::HRESULT) {
```

Comment:
The last use of this function was deleted in https://github.com/GuillaumeGomez/sysinfo/pull/1316.